notebooks/RohnPhenotypeClustering.ipynb

```diff
@@ -4,9 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Exemple of phenotypic clustering\n",
+    "# Example of phenotypic clustering\n",
     "\n",
-    "This notebook examplifies some steps involved in working with annotations from the IDR. It will build a figure similar to the figure 1.b of the paper from Rohn et al. \"Comparative RNAi screening identifies a conserved core metazoan actinome by phenotype\" (IDR008)."
+    "This notebook exemplifies some steps involved in working with annotations from the IDR. It will build a figure similar to the Figure 1.b of the paper from Rohn et al. \"Comparative RNAi screening identifies a conserved core metazoan actinome by phenotype\" (IDR008)."
```

Comment:
I think this is referring to Figure 1.b in Rohn et al's paper though.  So its ok here.